utils/build-script-impl

```diff
@@ -1184,6 +1197,9 @@ function build_directory_bin() {
             foundation)
                 echo "${root}/${FOUNDATION_BUILD_TYPE}/bin"
                 ;;
+            libdispatch)
+                echo "${root}/${LIBDISPATCH_BUILD_TYPE}/bin"
```

Comment:
libdispatch does not use CMake with Xcode generator.  This should return `"${root}/bin"`, if that's the correct binary directory for libdispatch.